cost/aws/old_snapshots/aws_delete_old_snapshots.pt

```diff
@@ -446,28 +440,29 @@ script "js_snapshots_cost_mapping", type: "javascript" do
         accountId: ds_get_caller_identity[0]['account'],
         snapshotId : snapshot['snapshotId'],
         region : snapshot['region'],
-        tagKeyValue : snapshot['tagKeyValue'],
+        tags : snapshot['tagKeyValue'],
         daysOld : snapshot['daysOld'],
         imageId : snapshot['imageId'],
         snapshotImageIdFormat : snapshot['snapshotImageIdFormat'],
-        savings : cur + ' '+formatNumber(monthly_savings.toFixed(3), separator),
+        savings : monthly_savings.toFixed(3),
```

Comment:
I'm seeing the incident having a string `savings` field instead of a number:
```
      "savings": "0.237",
```
In these policies, can we keep it as a number instead?
```
      "savings": 0.237,
```

Maybe we could skip calling `toFixed`?
```suggestion
        savings : monthly_savings,
```
 Or otherwise parse the string back to a number?:
```suggestion
        savings : Number(monthly_savings.toFixed(3)),
```